bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -64,30 +52,13 @@
 		</state>
 	</channel-type>
 
-	<channel-type id="power">
-		<item-type>Switch</item-type>
-		<label>Power</label>
-		<description>Switch the Plugwise Smart plug ON or OFF</description>
-		<category>switch</category>
-	</channel-type>
-
 	<channel-type id="lock">
 		<item-type>Switch</item-type>
 		<label>Lock</label>
 		<description>Locks the switch state of the Plugwise Smart plug</description>
 		<category>switch</category>
 	</channel-type>
 
-	<channel-type id="powerUsage">
-		<item-type>Number:Power</item-type>
-		<label>Power Usage</label>
-		<tags>
-			<tag>Measurement</tag>
-			<tag>Power</tag>
-		</tags>
-		<state pattern="%.2f %unit%" readOnly="true"></state>
```

Comment:
Not a request for change, just for the records: The system channel type now used has a state pattern with one decimal only. This is probably better, and can be configured by the user anyway.